components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -386,6 +403,77 @@ void AdsServiceImpl::Start() {
   BackgroundHelper::GetInstance()->AddObserver(this);
 }
 
+void AdsServiceImpl::MaybeShowFirstLaunchNotification() {
+  if (!ShouldShowAdsNotification() ||
+      !profile_->GetPrefs()->GetBoolean(
+          brave_rewards::prefs::kBraveRewardsEnabled))
+    return;
+
+  if (profile_->GetPrefs()->GetTime(
+          prefs::kBraveAdsLaunchNotificationTimestamp) == base::Time()) {
+      profile_->GetPrefs()->SetTime(
+          prefs::kBraveAdsLaunchNotificationTimestamp,
+        base::Time::Now());
```

Comment:
Should be indented 4 spaces however would fit on line 415

Reply:
fixed